spec/spec_helper_acceptance.rb

```diff
@@ -40,6 +40,12 @@
       shell 'apt-get install -y ssl-cert'
     end
 
+    if fact('operatingsystem') == 'SLES'
+      #generate certs for SLES
+      shell 'zypper install -y apache2'
```

Comment:
hmm... is this what we want to do here? Doesn't this circumvent the module installing apache using the platform provider?